README.md

```diff
@@ -1 +1,200 @@
-# nats-source-go
+# Nats Source
+Nats Source is a user-defined source for [Numaflow](https://numaflow.numaproj.io/)
+that facilitates reading messages from a Nats server.
+
+- [Quick Start](#Quick-Start)
+- [Using Nats Source in Your Numaflow Pipeline](#how-to-use-the-nats-source-in-our-own-numaflow-pipeline)
+- [JSON Configuration](#using-json-format-to-specify-the-nats-source-configuration)
+
```

Comment:
nitpick - consider capitalizing "NATS"
Also, would it be helpful to add a link to their site here?